### PR TITLE
Start of webpage

### PR DESCRIPTION
**Pull request by WendyKopf**
Just the beginning of the webpages. Will make more improvements over next iterations.

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+	return render_template('index.html')
+
+# Route for handling the login page logic
+@app.route('/login', methods=['GET', 'POST'])
+def login():
+	return render_template('login.html')
+	
+@app.route('/user/<name>')
+def user(name):
+  return render_template('user.html', name=name)
```
Comment (m-kopf) [inline app.py]: the indentation level here is different than the rest of the file

File: templates/index.html [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+    width: 100%;
+    margin: 0;
+    padding: 14px;
+  }
+  .topnav input[type=text] {
+    border: 1px solid #ccc;  
+  }
+}
+</style>
+</head>
```
Comment (m-kopf) [inline templates/index.html]: there's no <head> above to match with this

File: app.py (hunk 1 above)
Comment (WendyKopf) [reply app.py]: Done

File: templates/index.html (hunk 2 above)
Comment (WendyKopf) [reply templates/index.html]: Done

**Review by m-kopf — APPROVED** [review]

**Pull request merged**